### PR TITLE
add alt attribute to cluster image

### PR DESCRIPTION
**Pull request by Muskos**
I added `alt` attribute to the cluster.

### CODE REVIEW

**Comment by JustFly1984** [comment]
thank you, I will try to find time to release it next week.

File: packages/react-google-maps-api-marker-clusterer/src/ClusterIcon.tsx [hunk 1]
```diff
@@ -233,9 +233,17 @@ export class ClusterIcon {
 
       const pos = this.getPosFromLatLng(this.center)
 
+      // @ts-ignore
```
Comment (JustFly1984) [inline packages/react-google-maps-api-marker-clusterer/src/ClusterIcon.tsx]: please do not use @ts-ignore

File: packages/react-google-maps-api-marker-clusterer/src/ClusterIcon.tsx [hunk 2]
```diff
@@ -233,9 +233,17 @@ export class ClusterIcon {
...
+      if (typeof this.sums.title === "undefined" || this.sums.title === "") {
+        divTitle = this.cluster.getClusterer().getTitle()
+      } else {
+        // @ts-ignore
```
Comment (JustFly1984) [inline packages/react-google-maps-api-marker-clusterer/src/ClusterIcon.tsx]: please do not use @ts-ignore

**Comment by JustFly1984** [comment]
@Muskos can you please review my comments?

**Comment by Muskos** [comment]
@JustFly1984 updated

**Comment by JustFly1984** [comment]
there is build errors on netlify https://app.netlify.com/sites/react-google-maps-api-docs/deploys/5d710bc9823fae000d65cf08

Have you tried to build marker clusterer before pushing?
please be sure that it builds locally

**Comment by Muskos** [comment]
Ooops, closed. I will reopen.
Could you help me?
Screen Shot 2019-09-13 at 11 59 53 AM
Do you have a problem with `tsconfig.json`? Cause I haven't any tsconfig.json in root folder.

**Comment by JustFly1984** [comment]
Sorry all the files has disappeared from PR. We have no issues with tsconfig, each package has its own tsconfig, no root tsconfig

**Pull request closed without merging**